docs/chapter2-payload-image-format.rst

```diff
@@ -0,0 +1,659 @@
+   ``timestamp``           R     ``<u32>``             Last image modification time, as seconds
+                                                       in POSIX time format\ :sup:`1`. This is updated by
+                                                       any tool which creates or changes the FIT.
+   ``size``                OR    ``<u32>``             Specifies current serial device speed in bits
+                                                       per second
+   ``align``               R     ``<u32>``             Required alignment for images. Each image
+                                                       in the FIT is aligned to this value
+                                                       ('data-offset' property). Platform Init
+                                                       must ensure that each image is loaded to
+                                                       an address with this alignment, if no
+                                                       fixed load address is specified by the
+                                                       image.
+   ``spec-version``        OR    ``<u32>``             UPL image specification version in BCD
+                                                       format
+                                                       7 : 0 - Minor Version
+                                                       15 : 8 - Major Version
+                                                       31: 16 - Reserved
+                                                       For revision v0.90, the value will be
+                                                       0x0090.
+   ``build-version``       O     ``<u32>``             Payload build revision.
+                                                       Major.Minor.Revision.Build
+                                                       The ImageRevision can be decoded as
+                                                       follows:
+                                                       7 : 0  - Build Number
+                                                       15 :8  - Revision
+                                                       23 :16 - Minor Version
+                                                       31 :24 - Major Version
+   ======================= ===== ===================== ===============================================
+
+.. note:: 
+
+   1. When we get closer to 2106 we can consider allowing 64-bit values.
+
+**Node: /images/<name>**
+
+.. tabularcolumns:: | p{4cm} p{0.75cm} p{4cm} p{6.5cm} |
+.. table:: Node: /images/<name>
+
+   ================= =========== ============ ==============================================
+   Property Name     Usage       Value Type   Definition
+   ================= =========== ============ ==============================================
+   description       R           string       General description of the Loader. This may
```

Comment:
Change "Loader" to "image"? Each image could have its description.

Reply:
Done.